### ISSUE TITLE
How do I make fullPage.js allow scrolling down a webpage when it gets to the last slide?

### ISSUE THREAD

**Issue by LiamKenneth** [alvarotrigo/fullPage.js#2458]
Hi, wondering how do I make a header with fullPage.js but allow scrolling down a webpage when a user gets to the last slide?

I also would like to keep the smooth scrolling too until the user scrolls past the last slide and on to the standard website underneath. 

Thanks

**Comment by alvarotrigo** [comment]
Not possible.

**Comment by andrade1379** [comment]
Hello, just came across this plugin... would like to use and be allowed to scroll past the container where fullpage is taking over.

I've set scrollOverFlow and I still can't scroll beyond the set container.

Here's a screenshot of my pages structure. The first section and last two sections in the div with class of "snap" AND I also noticed that the `fitToSection: false` does nothing

<img width="767" alt="screen shot 2018-02-21 at 4 09 54 pm" src="https://user-images.githubusercontent.com/1234484/36508264-d41df7d0-1721-11e8-8fca-aef15f61afdf.png">
